draft-law-moq-cat4moqt.md

```diff
@@ -186,19 +212,65 @@ Prohibits
 
 ### Multiple actions
 
-Multiple actions may be communicated within the same token, with different permissions. The order
-in which Action/Permission tuples are declared and evaluated is unimportant. The evaluation stops
-after the first Permitted result is discovered.
+Multiple actions may be communicated within the same token, with different
+permissions. This can be facilitated by the logical claims defined in
+{draft-lemmons-composite-claims} or simply by defining multiple limits,
+depending on the required restrictions. In both cases, the order in which
+limits are declared and evaluated is unimportant. The evaluation stops after
+the first acceptable result is discovered.
 
 #### Example of evaluating multiple actions in the same token:
 
+~~~~~~~~~~~~~~~
+{
+    /moqt/ TBD_MOQT: [
+        [/PUBLISH/ 2, { /exact/ 0: "example.com"}, { /prefix/ 1: "bob"}],
+        [/PUBLISH/ 2, { /exact/ 0: "example.com"}, { /exact/ 0: "logs/12345/bob"}]
+    ],
+    /exp/ 4: 1750000000
+}
+~~~~~~~~~~~~~~~
+
 * (1) PUBLISH (Allow with a prefix match) example.com/bob
 * (2) PUBLISH (Allow with an exact match) example.com/logs/12345/bob
 
 Evaluating "example.com/bob/123" would succeed on test 1 and test 2 would never be evaluated.
 Evaluating "example.com/logs/12345/bob" would fail on test 1 but then succeed on test 2.
 Evaluating "example.com" would fail on test 1 and on test 2.
 
+In addition, the entire token expires at 2025-05-02T21:57:24+00:00.
+
+#### Example of evaluating multiple actions with related claims:
+
+If there are other claims that depend on which MOQT limit applies, a logical claim is required:
+
+~~~~~~~~~~~~~~~
+{
+    /or/ TBD_OR: [
+        {
+            /moqt/ TBD_MOQT: [[/PUBLISH/ 2, { /exact/ 0: "example.com"}, { /prefix/ 1: "bob"}]],
+            /exp/ 4: 1750000000
+        },
+        {
+            /moqt/ TBD_MOQT: [[/PUBLISH/ 2, { /exact/ 0: "example.com"}, { /exact/ 0: "logs/12345/bob"}]],
+            /exp/ 4: 1750000600
```

Comment:
Would be nice to add an additional example showing a complete set of composite claims, using expiry, not before, cwt ID, geo, etc.

Reply:
Yeah, once we're happy with the claim itself, a couple of fulsome examples are definitely in order.